tasks/ipv6.yml

```diff
@@ -35,7 +35,8 @@
     value: "{{ item.value|int }}"
     state: present
     sysctl_set: 'yes'
-  with_dict: "{{ ipv6_sysctl_settings }}"
+  when: system_has_ipv6
```

Comment:
`when: system_has_ipv6 and disable_ipv6`

Reply:
Done. In unrelated code I ran into the fact that saying. `set_fact: myvarname` doesn't actually evaluate `myvarname` but just takes it as string. It needs to be `set_fact: "{{ myvarname }}"`

This was an error in the code I introduced in facts.yml, has been fixed.